### ISSUE TITLE
Any plans for supporting replicant OS?

### ISSUE THREAD

**Issue by lerdem** [TeamNewPipe/NewPipe#8158]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

Any plans for supporting replicant OS?

**Comment by SameenAhnaf** [comment]
Please keep the discussion in https://github.com/TeamNewPipe/NewPipe/issues/1051

**Comment by mhmdanas** [comment]
@lerdem Replicant is a custom Android ROM, so it _should_ be able to run NewPipe. Is that not the case?

**Comment by lerdem** [comment]
In general app works. I have issues with playing videos and audio. Internal newpipe player does not play video, app is crashing. Same situation with internal audio player. So for now i am using newpipe for searching and playing audio with external player(vlc). Video playing with external player(vlc) work olny with quality 144p and very slow(but its no newpipe issue, just for illustration of whole picture).

So, for all of that, i asked about supporting replicant OS. 
Perhaps system is to old, and does not support some proprietary codes(or something else). That is why newpipe team does not _fully_ support replicant OS.

Tested on phone [Galaxy S III (GT-I9300)](https://redmine.replicant.us/projects/replicant/wiki/GalaxySIIIGTI9300)

**Comment by litetex** [comment]
> Internal newpipe player does not play video, app is crashing. Same situation with internal audio player.

Crash report pls.

I'm not sure if a galaxy S3 from 2013 is working with a new OS. There might be hardware features missing.

**Comment by moriel5** [comment]
lerdem Could you please test playback using hardware acceleration only? Such as with the AOSP Gallery or with MiXplorer's video player?

If this is due to missing codecs, then this has nothing to do with whether NewPipe supports the ROM or not, since it relies on the system libraries to playback content. As such, the situation would be that the ROM does not support the codecs, not that NewPipe does not support the ROM.

VLC, by default, plays back content with hardware acceleration, unless not supported by the ROM or hardware, in which case it switches to software acceleration, which probably why it is so slow.

Also, you have not reported enough information for anything to be gleaned from your post.

The devs need the following from you:

• Device
• OS Version (not just which ROM it is)
• NewPipe Version

**Comment by lerdem** [comment]
I have created bug-report #8181. Suppose it is actually a bug, because `skyTube` app video playing feature works fine(on the same replicant OS device).

**Comment by moriel5** [comment]
That's great, although from my experience posting bugs, it's generally a better idea to edit the OP with the information, rather than creating a new issue, since it reduces clutter for the devs.

I hope that the root of this issue is found, with the issue itself also being resolved.

**Comment by litetex** [comment]
See explanation from https://github.com/TeamNewPipe/NewPipe/issues/8181#issuecomment-1097145945.

TL;DR
If you use a OS that disables the GPU apps that require advanced rendering processes don't work anymore.
